website/pages/en/index.js

```diff
@@ -138,17 +138,17 @@ class Index extends React.Component {
                 <div className="blockImage">
                   <div>
                     <MarkdownBlock>
-                      {typeAnnotationExample}
+                      {asyncExample}
                     </MarkdownBlock>
                   </div>
                 </div>
                 <div className="blockContent">
                   <h2>
-                    Type Annotations
+                    Async
                   </h2>
                   <div>
                     <MarkdownBlock>
-                      Type annotations allow for code to be explicitly typed on parameters, class member variables and return values.
+                      **[Asynchronous operations](https://docs.hhvm.com/hack/asynchronous-operations/introduction)** allow _cooperative multi-tasking_. Code that utilizes the Async Infra can hide I/O latency and data fetching, minimizing the downtime a program has to be stalled for I/O or similar operations.
```

Comment:
I might avoid "downtime" as async doesn't affect robustness. It just makes your program faster.

```suggestion
                      **[Asynchronous operations](https://docs.hhvm.com/hack/asynchronous-operations/introduction)** allow _cooperative multi-tasking_. Async functions can combine I/O requests, making your code faster.
```